### ISSUE TITLE
Translation tool confirmation message

### ISSUE THREAD

**Issue by xavismeh** [codecombat/codecombat#3322]
When translating anything from the tool (for instance https://codecombat.com/i18n/level/wakka-maul), a confirmation message should be displayed to ensure the modifications have been saved.

**Comment by blorente** [comment]
Right now there are some achievement messages appearing the first few times you save, I assume you mean another, more consistent message liks "Changes saved"?

**Comment by xavismeh** [comment]
yeah exactly. Ideally, it should remain displayed for a few seconds to get the information, and be displayed again when you save an other time. I personally do it a lot, especially when I work on a level translation to don't lose any changes. Thx

**Comment by blorente** [comment]
Perfect, I'll try and fix it this week.

**Comment by xavismeh** [comment]
:+1:
